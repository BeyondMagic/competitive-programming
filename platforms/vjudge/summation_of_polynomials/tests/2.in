27324
32853
19983
20007
25776
46931
36565
12103
14964
22871
20197
25596
16087
14688
27856
8598
17402
26465
1
19472
5320
19084
3745
2268
49231
10409
43305
20
11056
35479
301
38380
18331
16494
8386
44106
13424
50000
22561
28388
3417
42757
3983
16611
237
31839
25208
411
24655
8872
