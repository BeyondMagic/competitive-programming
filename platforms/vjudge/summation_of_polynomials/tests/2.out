139363454590222500
291250434306365361
39868163210258496
40060033708400784
110366125849728576
1212823906893859716
446916175935231025
5365175160990736
12536862135696900
68409844171195536
41603558154067009
107315474103244836
16745350701053584
11637234828032256
150538021755351616
1366566556344201
22929077260791009
122647986166709025
1
35944037455800384
200331752899600
33163713376264900
49201596928225
6620565718116
1468629614327127616
2935347245534025
879249717924417225
44100
3736032864625216
396141162654451600
2065793401
542478488455332100
28231374337626916
18505321254090225
1236696155889481
946129086832772241
8119559793960000
1562562500625000000
64775666382230881
162371495761413156
34101547160409
835582289068221409
62950514066496
19035987458973156
795409209
256924171478534400
100955044726503696
7168331556
92383681545985600
1549259036554384
